### PR TITLE
adding design documentation to README

### PR DESCRIPTION
**Pull request by heatherlarsen**
## :pushpin: Summary

If merged, this PR adds consumer and contribution instructions for designers to the README

### :speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

Examples: 
- issue (ux,non-blocking): These buttons should be red, but let's handle this in a follow-up.
- suggestion (non-blocking): Let's change this wording to make it easier to understand.
- issue (blocking): We shouldn't introduce this kind of tech debt; let's pair and resolve the issue in a more sustainable way.

## 👉 Next steps
* Add design guidance (ie. icon types, when to use certain icons, icon geometry, etc)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,10 +28,38 @@ Please see [ember-flight-icons/README](ember-flight-icons/README.md). This addon
...
+
+💡 Tip: Swapping instances, sizes, and colors is easy to do from the right sidebar
+
 ## Contributing
 
 See the [Contributing](CONTRIBUTING.md) guide for details.
 
+### How to contribute a new icon in Figma (HashiCorp staff only)
+
+1. [Open a GitHub issue](https://github.com/hashicorp/flight/issues/new) to kick-off work on adding a new icon
```
Comment (Dhaulagiri) [inline README.md:47]: Opened https://github.com/hashicorp/flight/issues/148 to allow us to better formalize this

**Review by MelSumner — APPROVED** [review]
Thank you for adding this!

**Comment by MelSumner** [comment]
ok @heatherlarsen I think I managed to move everything to the template. Please look at #157 and LMK if you want any tweaks (dm on slack is fine)

File: README.md [hunk 2]
```diff
@@ -28,10 +28,82 @@ Please see [ember-flight-icons/README](ember-flight-icons/README.md). This addon
...
+
+Some common action icons that get used within the products include:  
+
+![example of common actions](ember-flight-icons/tests/dummy/public/assets/images/icon-actions.png)
+
 ## Contributing
 
 See the [Contributing](CONTRIBUTING.md) guide for details.
 
+### How to contribute a new icon in Figma (HashiCorp staff only)
```
Comment (amyrlam) [inline README.md:90]: Still catching up, looks like this was all brought in as part of https://github.com/hashicorp/flight/pull/157/files except for this section "How to contribute a new icon in Figma (HashiCorp staff only)" do you still want to merge that part in? @heatherlarsen @MelSumner

**Comment by amyrlam** [comment]
it seems like this can be closed as https://github.com/hashicorp/flight/issues/175 was filed and most of it was merged in. please reopen if i'm wrong!

**Pull request closed without merging**